README.md

```diff
@@ -305,12 +305,9 @@ $ drive pull --files a1/b2
 $ drive pull --directories tf1
 ```
 
-## Note: Checksum verification:
+Due to popular demand, by default checksum verification is turned off. It was deemed to be quite vigorous and unnecessary for most cases, in which size + modTime differences are sufficient to detect file changes. The discussion stemmed from issue [#117](https://github.com/odeke-em/drive/issues/117).
```

Comment:
Please add a comma to `by default checksum` -> `by default, checksum`.

I think that not having that header for that note will make it harder to look for information on checksum verification. The reason I mention this is because even with that sub-heading, people didn't seem to find information about checksum verification in drive and seem to have to ask questions about it https://www.reddit.com/r/linux/comments/2u7224/official_google_drive_linux_client_screenshots/co5w192 and https://www.reddit.com/r/linux/comments/2u7224/official_google_drive_linux_client_screenshots/co6q77a